etna/datasets/utils.py

```diff
@@ -225,15 +225,11 @@ def get_level_dataframe(
     :
        dataframe at the target level
```

Comment:
At parameters section you write that `source_level_segments` is tuple, but in typing it is `List`.